Test__Helper_MiscFso.Test_GetAllFilesRecursive fails when repo not in `E:\AA\MiscVBAFunctions`

Test__Helper_MiscFso.Test_GetAllFilesRecursive

```
PS C:\aa\MiscVBAFunctions> & C:/Users/malan/anaconda3/python.exe c:/aa/MiscVBAFunctions/tests/testMiscFso.py
F
======================================================================
FAIL: test_1 (__main__.TestDictsToTable)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\aa\MiscVBAFunctions\tests\testMiscFso.py", line 16, in test_1
    self.assertTrue(
AssertionError: False is not true

----------------------------------------------------------------------
Ran 1 test in 1.384s

FAILED (failures=1)
```


The actual test function:

```
Function Test_GetAllFilesRecursive(InputPath As String)
    Dim AllFiles As Collection
    Dim Pass As Boolean
    Pass = True
    
    Set AllFiles = GetAllFilesRecursive(Fso.GetFolder(InputPath))

    Pass = 5 = CInt(AllFiles.Count) = Pass = True
    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\empty file.txt" = AllFiles(1) = Pass = True
    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\folder1\empty file.txt" = AllFiles(2) = Pass = True
    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\folder1\folder1\empty file.xlsx" = AllFiles(3) = Pass = True
    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\folder2\empty file.docx" = AllFiles(4) = Pass = True
    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\folder2\folder1\folder1\empty file.txt" = AllFiles(5) = Pass = True

    Test_GetAllFilesRecursive = Pass
End Function
```
These should test/assert relative to the repo and not the full path.

--- a/MiscVBAFunctions/Modules/Test__Helper_MiscFso.bas
+++ b/MiscVBAFunctions/Modules/Test__Helper_MiscFso.bas
@@ -2,18 +2,10 @@ Attribute VB_Name = "Test__Helper_MiscFso"
 Option Explicit
 
 Function Test_GetAllFilesRecursive(InputPath As String)
-    Dim AllFiles As Collection
-    Dim Pass As Boolean
-    Pass = True
+    Set Test_GetAllFilesRecursive = New Collection
     
-    Set AllFiles = GetAllFilesRecursive(Fso.GetFolder(InputPath))
-
-    Pass = 5 = CInt(AllFiles.Count) = Pass = True
-    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\empty file.txt" = AllFiles(1) = Pass = True
-    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\folder1\empty file.txt" = AllFiles(2) = Pass = True
-    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\folder1\folder1\empty file.xlsx" = AllFiles(3) = Pass = True
-    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\folder2\empty file.docx" = AllFiles(4) = Pass = True
-    Pass = "E:\AA\MiscVBAFunctions\test_data\GetAllFiles\folder2\folder1\folder1\empty file.txt" = AllFiles(5) = Pass = True
-
-    Test_GetAllFilesRecursive = Pass
+    Dim F As File
+    For Each F In GetAllFilesRecursive(Fso.GetFolder(InputPath))
+        Test_GetAllFilesRecursive.Add F.Path
+    Next F
 End Function

--- a/tests/testMiscFso.py
+++ b/tests/testMiscFso.py
@@ -11,10 +11,22 @@ class TestDictsToTable(unittest.TestCase):
     def test_1(self) -> None:
         book: Book
         with functions_book() as book:
-            func = book.macro("Test__Helper_MiscFso.Test_GetAllFilesRecursive")
+            look_in_path = Path(this_dir().parent, r"test_data\GetAllFiles")
 
-            self.assertTrue(
-                func(str(Path(this_dir().parent, r"test_data\GetAllFiles").absolute()))
+            self.assertEqual(
+                [
+                    "empty file.txt",
+                    "folder1/empty file.txt",
+                    "folder1/folder1/empty file.xlsx",
+                    "folder2/empty file.docx",
+                    "folder2/folder1/folder1/empty file.txt",
+                ],
+                [
+                    Path(p).relative_to(look_in_path).as_posix()
+                    for p in book.macro(
+                        "Test__Helper_MiscFso.Test_GetAllFilesRecursive"
+                    )(str(look_in_path.absolute()))
+                ],
             )
 
 

